Update ApiVersion to 2016-07-07 and Introduce Support for VNET

solankisamir opened the pull request:
- Bumped up the ApiVersion to 2016-07-07 for RP and SMAPI (Management API)
- Introduced Support for ARM VNET
- Introduced Support for Internal Facing Workloads
- Deprecated api ManageVirtualNetwork
- Changed api for CheckServiceNameAvailability
- New tests for Importing Swagger Api
- Deprecated Enable Tenant Git Access

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

Commit:
Iteration 4

Commit:
Updated generated hyak specs once again

peshen committed:
Release Cassandra SDK

peshen committed:
Release Salesforce SDK

amitbapat committed:
Updated readme.md to fix formatting issues (#2229)

Commit:
paging support

Commit:
Incremented nuget and assembly version inorder to be able to publish

peshen committed:
Add Sample test

Add Sample test

yoavrubin committed:
Adding SQL threat detection server level and properties for blob auditing

yoavrubin committed:
Adding recorded test runs

yoavrubin committed:
Updating project to copy new test

avneeshraiusit committed:
1) Changes to support A2A operations.
2) V2A scenario fixes.
3) HUB changes
4) B2A Exclude Disk Contract Changes.
5) Changes for job status event

avneeshraiusit committed:
HUB changes

Commit:
Added test for ADJoined Clusters

Commit:
Moved SecurityProfile to ClusterCreateParameters

Commit:
updated Version and resolved CR comments

avneeshraiusit committed:
Taking review comments: Added documentation for new fields.

avneeshraiusit committed:
Readding create and remove site tests.

Commit:
Added Test record in csproj

zhaodong2013062 committed:
Add MultipleWebServiceInputs to AzureMLBatchExecutionActivity

  * Enable passing multiple web service inputs to an Azure Machine
    Learning experiment

Commit:
Version bump to 1.2.0

avneeshraiusit committed:
Adding tests for new scenarios and change to test constants.

Commit:
reverted the AssembleVersion to 1.0.0.0

atpham256 committed:
Add support for the OutputErroryPolicy property

-Added generated files necessary to support the OutputErrorPolicy
property at the ASA job level
-Updated tests to test the new property
-Reran tests and updated session records
-Updated NuGet version to 1.8.1

atpham256 committed:
Update session record

- Added missing session record update

vivsriaus committed:
Remove location as required property for Resource

vivsriaus committed:
Add a test to ensure you can create a resource without location

vivsriaus committed:
Add customization folder to avoid breaking change

vivsriaus committed:
Update version for ARM assembly

Live2Learn committed:
Add three optional properties to TextFormat: SkipLineCount, FirstRowAsHeader and TreatEmptyAsNull

bgold09 committed:
Fixing package version number

  * Version 4.10.0 was never shipped, so reverted back to this version
    in order to publish the correct package version.
  * Modifying the changelog to match the correct version number.

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

solankisamir committed:
Update ApiVersion to 2016-07-07 and Introduce Support for VNET

- Bumped up the ApiVersion to 2016-07-07 for RP and SMAPI (Management
API)
- Introduced Support for ARM VNET
- Introduced Support for Internal Facing Workloads
- Deprecated api ManageVirtualNetwork
- Changed api for CheckServiceNameAvailability
- New tests for Importing Swagger Api
- Deprecated Enable Tenant Git Access

solankisamir committed:
Update ApiVersion to 2016-07-07 and Introduce Support for VNET

- Bumped up the ApiVersion to 2016-07-07 for RP and SMAPI (Management
API)
- Introduced Support for ARM VNET
- Introduced Support for Internal Facing Workloads
- Deprecated api ManageVirtualNetwork
- Changed api for CheckServiceNameAvailability
- New tests for Importing Swagger Api
- Deprecated Enable Tenant Git Access

solankisamir committed:
Update ApiVersion to 2016-07-07 and Introduce Support for VNET and Soap Api

- Bumped up the ApiVersion to 2016-07-07 for RP and SMAPI (Management
API)
- Introduced Support for ARM VNET
- Introduced Support for Internal Facing Workloads
- Deprecated api ManageVirtualNetwork
- Changed api for CheckServiceNameAvailability
- New tests for Importing Swagger Api
- Deprecated Enable Tenant Git Access

azurecla commented:
Hi __@solankisamir__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sasolank). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

solankisamir committed:
Fixed test by using the CopyIfNewer flag and Bumped Version to 2.0.4

Tests were failing, as the CopyIfNewer flag was not set for some records
and bumped nuget version to 2.0.4

solankisamir committed:
Add test for Importing Wsdl and Bump up Minor Version as it is Api Version Change

- Add test for testing Wsdl (Soap Api)
- Bump up the minor Version as it is an Api-Version change.

solankisamir committed:
Bump up Major Version as we are Deprecating Endpoint ManageVpn

- Bumped up Major Version.

shahabhijeet commented:
@solankisamir seems like a breaking change. Can you bump up the major version.
Also can you squash your merge commits and then push it.

solankisamir committed:
Merge remote-tracking branch 'refs/remotes/Azure/master'

solankisamir committed:
resolve merge conflicts

solankisamir committed:
Fix issue with Duplicated Files in Sql project causing build failure

solankisamir committed:
remove duplicate file copy in sql.test

solankisamir committed:
additional validation and minor code changes

KedarJoshi commented:
LGTM
